<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemAdminWidget</class>
 <widget class="QWidget" name="SystemAdminWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1124</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SystemAdminWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_18">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QWidget" name="title_widget" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>88</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>88</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="system_management_pushButton">
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>88</height>
           </size>
          </property>
          <property name="toolTip">
           <string>档案检索</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	border-radius:0px;
	background-image: url(:/pic/resource/img/titlebar/xitong_mo.png);
}
QPushButton#pushButton_usermanagement:hover{
	background-image: url(:/pic/resource/img/titlebar/xitong_hua.png);
}
QPushButton#pushButton_usermanagement:pressed{
	background-image: url(:/pic/resource/img/titlebar/xitong_en.png);
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_9">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>111</width>
            <height>76</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QLabel" name="label_15">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="styleSheet">
               <string notr="true">font: 10pt &quot;微软雅黑&quot;;
color: rgb(255, 0, 0);</string>
              </property>
              <property name="text">
               <string>上一次登录时间为：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lastlogintime_label">
              <property name="styleSheet">
               <string notr="true">font: 10pt &quot;微软雅黑&quot;;
color: rgb(255, 0, 0);</string>
              </property>
              <property name="text">
               <string>2019-11-07 11:30:33</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_17">
            <item>
             <widget class="QLabel" name="label_19">
              <property name="styleSheet">
               <string notr="true">font: 10pt &quot;微软雅黑&quot;;
color: rgb(255, 0, 0);</string>
              </property>
              <property name="text">
               <string>上次登录的ip地址为：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lastauth_ipaddress_label">
              <property name="styleSheet">
               <string notr="true">font: 10pt &quot;微软雅黑&quot;;
color: rgb(255, 0, 0);</string>
              </property>
              <property name="text">
               <string>ipaddress</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_32">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>6</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <property name="spacing">
               <number>15</number>
              </property>
              <item>
               <widget class="QLabel" name="icon_label">
                <property name="minimumSize">
                 <size>
                  <width>40</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>40</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-image: url(:/pic/resource/img/systemadmin.40.png);
</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_18">
                <property name="maximumSize">
                 <size>
                  <width>65</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">font: 14pt &quot;微软雅黑&quot;;
color: rgb(0, 0, 0);</string>
                </property>
                <property name="text">
                 <string>欢迎您：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="username_label">
                <property name="styleSheet">
                 <string notr="true">font: 14pt &quot;微软雅黑&quot;;
color: rgb(0, 0, 0);</string>
                </property>
                <property name="text">
                 <string>username</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_13">
              <property name="spacing">
               <number>15</number>
              </property>
              <item>
               <widget class="QLabel" name="label_12">
                <property name="styleSheet">
                 <string notr="true">font: 14pt &quot;微软雅黑&quot;;
color: rgb(0, 0, 0);</string>
                </property>
                <property name="text">
                 <string>系统管理员</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_16">
                <property name="styleSheet">
                 <string notr="true">font: 14pt &quot;微软雅黑&quot;;
color: rgb(0, 0, 0);</string>
                </property>
                <property name="text">
                 <string>退出</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="logout_pushButton">
                <property name="minimumSize">
                 <size>
                  <width>28</width>
                  <height>28</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>28</width>
                  <height>28</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>退出账户</string>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton{
border-radius:14px;
border-image: url(:/pic/resource/img/logout_button_default.png);
} 
QPushButton:pressed{
border-radius:14px;
border-image: url(:/pic/resource/img/logout_button_pressed.png);
}
QPushButton:hover{
border-radius:14px;
border-image: url(:/pic/resource/img/logout_button_hover.png);
} </string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_19"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="main_stackedWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="system_management_page">
        <layout class="QHBoxLayout" name="horizontalLayout_27">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="statistic_widget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_21">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="statistic_horizontalLayout">
              <item>
               <widget class="QScrollArea" name="scrollArea">
                <property name="widgetResizable">
                 <bool>true</bool>
                </property>
                <widget class="QWidget" name="scrollAreaWidgetContents">
                 <property name="geometry">
                  <rect>
                   <x>0</x>
                   <y>0</y>
                   <width>1114</width>
                   <height>506</height>
                  </rect>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_33">
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_26">
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_24">
                      <item>
                       <layout class="QVBoxLayout" name="verticalLayout_13">
                        <item>
                         <layout class="QHBoxLayout" name="horizontalLayout_3">
                          <item>
                           <widget class="QLabel" name="label_3">
                            <property name="styleSheet">
                             <string notr="true">font: 14pt &quot;微软雅黑&quot;;
color: rgb(0, 0, 0);</string>
                            </property>
                            <property name="text">
                             <string>服务器状态：</string>
                            </property>
                           </widget>
                          </item>
                          <item>
                           <widget class="QLabel" name="label_4">
                            <property name="styleSheet">
                             <string notr="true">font: 14pt &quot;微软雅黑&quot;;
color: rgb(0, 0, 0);</string>
                            </property>
                            <property name="text">
                             <string>正常</string>
                            </property>
                           </widget>
                          </item>
                         </layout>
                        </item>
                        <item>
                         <layout class="QVBoxLayout" name="verticalLayout_11">
                          <item>
                           <widget class="QLabel" name="label_17">
                            <property name="styleSheet">
                             <string notr="true">font: 14pt &quot;微软雅黑&quot;;
color: rgb(0, 0, 0);</string>
                            </property>
                            <property name="text">
                             <string>服务器信息：</string>
                            </property>
                           </widget>
                          </item>
                          <item>
                           <widget class="QListWidget" name="listWidget_3">
                            <item>
                             <property name="text">
                              <string>系统CPU个数：</string>
                             </property>
                            </item>
                            <item>
                             <property name="text">
                              <string>JAVA版本：</string>
                             </property>
                            </item>
                            <item>
                             <property name="text">
                              <string>服务开始运行时间：2023-06-16 16:19:46</string>
                             </property>
                            </item>
                            <item>
                             <property name="text">
                              <string>端口：</string>
                             </property>
                            </item>
                            <item>
                             <property name="text">
                              <string>进程号：</string>
                             </property>
                            </item>
                           </widget>
                          </item>
                         </layout>
                        </item>
                       </layout>
                      </item>
                      <item>
                       <layout class="QVBoxLayout" name="verticalLayout_19">
                        <item>
                         <layout class="QHBoxLayout" name="horizontalLayout_4">
                          <item>
                           <widget class="QLabel" name="label_5">
                            <property name="styleSheet">
                             <string notr="true">font: 14pt &quot;微软雅黑&quot;;
color: rgb(0, 0, 0);</string>
                            </property>
                            <property name="text">
                             <string>数据库状态：</string>
                            </property>
                           </widget>
                          </item>
                          <item>
                           <widget class="QLabel" name="label_7">
                            <property name="styleSheet">
                             <string notr="true">font: 14pt &quot;微软雅黑&quot;;
color: rgb(0, 0, 0);</string>
                            </property>
                            <property name="text">
                             <string>正常</string>
                            </property>
                           </widget>
                          </item>
                         </layout>
                        </item>
                        <item>
                         <layout class="QVBoxLayout" name="verticalLayout_14">
                          <item>
                           <widget class="QLabel" name="label_20">
                            <property name="styleSheet">
                             <string notr="true">font: 14pt &quot;微软雅黑&quot;;
color: rgb(0, 0, 0);</string>
                            </property>
                            <property name="text">
                             <string>数据库信息：</string>
                            </property>
                           </widget>
                          </item>
                          <item>
                           <widget class="QListWidget" name="listWidget_4">
                            <item>
                             <property name="text">
                              <string>数据库类型：PostgreSQL</string>
                             </property>
                            </item>
                            <item>
                             <property name="text">
                              <string>JDBC连接最大数：10</string>
                             </property>
                            </item>
                            <item>
                             <property name="text">
                              <string>JDBC连接最小数：10</string>
                             </property>
                            </item>
                           </widget>
                          </item>
                         </layout>
                        </item>
                       </layout>
                      </item>
                      <item>
                       <layout class="QVBoxLayout" name="verticalLayout_2">
                        <item>
                         <layout class="QHBoxLayout" name="horizontalLayout_7">
                          <item>
                           <widget class="QLabel" name="label_9">
                            <property name="styleSheet">
                             <string notr="true">font: 14pt &quot;微软雅黑&quot;;
color: rgb(0, 0, 0);</string>
                            </property>
                            <property name="text">
                             <string>磁盘使用状况</string>
                            </property>
                           </widget>
                          </item>
                          <item>
                           <widget class="QProgressBar" name="progressBar">
                            <property name="value">
                             <number>0</number>
                            </property>
                           </widget>
                          </item>
                         </layout>
                        </item>
                        <item>
                         <widget class="QListWidget" name="listWidget">
                          <item>
                           <property name="text">
                            <string>磁盘总空间：</string>
                           </property>
                          </item>
                          <item>
                           <property name="text">
                            <string>磁盘已使用空间：</string>
                           </property>
                          </item>
                          <item>
                           <property name="text">
                            <string>磁盘未使用空间：</string>
                           </property>
                          </item>
                         </widget>
                        </item>
                       </layout>
                      </item>
                      <item>
                       <layout class="QVBoxLayout" name="verticalLayout_5">
                        <item>
                         <layout class="QHBoxLayout" name="horizontalLayout_8">
                          <item>
                           <widget class="QLabel" name="label_10">
                            <property name="styleSheet">
                             <string notr="true">font: 14pt &quot;微软雅黑&quot;;
color: rgb(0, 0, 0);</string>
                            </property>
                            <property name="text">
                             <string>内存占用：</string>
                            </property>
                           </widget>
                          </item>
                          <item>
                           <widget class="QProgressBar" name="progressBar_2">
                            <property name="value">
                             <number>0</number>
                            </property>
                           </widget>
                          </item>
                         </layout>
                        </item>
                        <item>
                         <widget class="QListWidget" name="listWidget_2">
                          <item>
                           <property name="text">
                            <string>JVM最大内存：</string>
                           </property>
                          </item>
                          <item>
                           <property name="text">
                            <string>JVM已使用内存大小: </string>
                           </property>
                          </item>
                          <item>
                           <property name="text">
                            <string>JVM可用内存大小：</string>
                           </property>
                          </item>
                         </widget>
                        </item>
                       </layout>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_28">
                      <item>
                       <layout class="QVBoxLayout" name="verticalLayout_20">
                        <item>
                         <widget class="QLabel" name="label_21">
                          <property name="styleSheet">
                           <string notr="true">font: 14pt &quot;微软雅黑&quot;;
color: rgb(0, 0, 0);</string>
                          </property>
                          <property name="text">
                           <string>最近的网络请求：</string>
                          </property>
                         </widget>
                        </item>
                        <item>
                         <widget class="QTableWidget" name="tableWidget_5">
                          <column>
                           <property name="text">
                            <string>请求ip</string>
                           </property>
                          </column>
                          <column>
                           <property name="text">
                            <string>请求时间</string>
                           </property>
                          </column>
                          <column>
                           <property name="text">
                            <string>返回状态信息</string>
                           </property>
                          </column>
                         </widget>
                        </item>
                       </layout>
                      </item>
                      <item>
                       <layout class="QVBoxLayout" name="verticalLayout_22">
                        <item>
                         <widget class="QLabel" name="label_23">
                          <property name="styleSheet">
                           <string notr="true">font: 14pt &quot;微软雅黑&quot;;
color: rgb(0, 0, 0);</string>
                          </property>
                          <property name="text">
                           <string>服务器日志信息：</string>
                          </property>
                         </widget>
                        </item>
                        <item>
                         <widget class="QListWidget" name="listWidget_7"/>
                        </item>
                       </layout>
                      </item>
                      <item>
                       <layout class="QVBoxLayout" name="verticalLayout_25">
                        <item>
                         <widget class="QLabel" name="label_26">
                          <property name="styleSheet">
                           <string notr="true">font: 14pt &quot;微软雅黑&quot;;
color: rgb(0, 0, 0);</string>
                          </property>
                          <property name="text">
                           <string>Mappings：</string>
                          </property>
                         </widget>
                        </item>
                        <item>
                         <widget class="QListWidget" name="listWidget_10"/>
                        </item>
                       </layout>
                      </item>
                      <item>
                       <layout class="QVBoxLayout" name="verticalLayout_21">
                        <item>
                         <widget class="QLabel" name="label_22">
                          <property name="styleSheet">
                           <string notr="true">font: 14pt &quot;微软雅黑&quot;;
color: rgb(0, 0, 0);</string>
                          </property>
                          <property name="text">
                           <string>自定义接口：</string>
                          </property>
                         </widget>
                        </item>
                        <item>
                         <widget class="QListWidget" name="listWidget_6">
                          <item>
                           <property name="text">
                            <string>name : myendponit</string>
                           </property>
                          </item>
                         </widget>
                        </item>
                       </layout>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
